python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -191,7 +191,11 @@ def accept(self, visitor):
         visitor.nodeBegin(self)
 
         for (name, item) in self._schema.simpleItems():
-            if item['visitable']:
+
+            # Special check for Ganga list to allow nested lists
+            if name == "_list" and self._name == "GangaList":
+                visitor.componentAttribute(self, "_list", self._getdata("_list"), 1)
```

Comment:
I feel that instead of having a special check, we should use inheritance and just override `accept` in `GangaList` to something like:
```python
def accept(self, visitor):
    visitor.nodeBegin(self)

    for (name, item) in self._schema.simpleItems():
        if name == '_list':
            visitor.componentAttribute(self, '_list', self._getdata('_list'), 1)
        elif item['visitable']:
            visitor.simpleAttribute(self, name, self._getdata(name), item['sequence'])

    for (name, item) in self._schema.componentItems():
        if item['visitable']:
            visitor.componentAttribute(self, name, self._getdata(name), item['sequence'])

    visitor.nodeEnd(self)
```
so that the `GangaList`-specific logic is contained within `GangaList`